DigitalLearningSolutions.Data/Services/ImageResizeService.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System;
+    using System.Drawing;
+    using System.Drawing.Drawing2D;
+    using System.Drawing.Imaging;
+    using System.IO;
+    using Microsoft.AspNetCore.Http;
+
+    public interface IImageResizeService
+    {
+        public byte[] ResizeProfilePicture(IFormFile formProfileImage);
+    }
+
+    public class ImageResizeService : IImageResizeService
+    {
+        public byte[] ResizeProfilePicture(IFormFile formProfileImage)
+        {
+            using var memoryStream = new MemoryStream();
+            formProfileImage.CopyTo(memoryStream);
+
+            return SquareImageFromMemoryStream(memoryStream, 300);
+        }
+
+        private byte[] SquareImageFromMemoryStream(MemoryStream memoryStream, int targetSideLength)
```

Comment:
Could we specify the units in this variable name - targetSideLengthInPx to make this a bit more concise. Then if someone wants to use this in the future, they don't need to dig into the implementation to find out :)